playground/mocks/idp/idx/data/factor-enroll-options.json

```diff
@@ -12,58 +12,61 @@
           "create-form"
         ],
         "name": "select-factor",
-        "href": "http://localhost:3000/idp/idx",
+        "href": "http://manumalhotra.okta1.com:1802/idp/idx/credential/enroll",
```

Comment:
I think we should keep these  as `localhost:3000` so that it works well with the playground ?